Dockerfile

```diff
@@ -31,7 +31,7 @@ RUN apk add --update --no-cache ca-certificates
 ##     docker build --no-cache -t vela-terraform:local .     ##
 ###############################################################
 
-FROM scratch
+FROM alpine:3.12.0
```

Comment:
I'm guessing the plugin doesn't work if we leave it with a `scratch` base? 😞

Reply:
Sadly, not from the testing I've seen. My experience running it in pipelines saw failures for Terraform to write to `/tmp/*`. You can manually add that dir in the scratch image but then you run int Terraform needing to write within the project for plugins, override, state, modules files, etc.

I was debating between Alpine and Busybox for the base image. Kinda settled on Alpine for the filesystem benefits. Busybox would have been more ideal since it's smaller but it didn't give me the filesystem benefits Alpine did.